app/src/main/res/values/styles.xml

```diff
@@ -14,6 +14,21 @@
         <item name="colorAccent">@color/colorAccent</item>
     </style>
 
+    <style name="AppTheme.TransparentActionBar" parent="AppTheme">
+        <item name="android:actionBarStyle">@style/TransparentActionBar</item>
+        <item name="android:windowActionBarOverlay">true</item>
+        <!-- Support library compatibility -->
+        <item name="actionBarStyle">@style/TransparentActionBar</item>
+        <item name="windowActionBarOverlay">true</item>
+    </style>
+
+    <style name="TransparentActionBar"
+        parent="Widget.AppCompat.Light.ActionBar.Solid">
+        <item name="android:background">#666666</item>
+        <!-- Support library compatibility -->
+        <item name="background">@color/colorHeavyGray</item>
+    </style>
+
```

Comment:
There's something still off with this styling, which one of us should probably work on. See  user story #90 here https://tree.taiga.io/project/krtonga-dawasco-app/us/90?milestone=145381